File: hw/raura/U2/hw13ToOurProgrammingLanguage/src/view/BasicOperations.js
const BasicOperation = require('../controller/BasicOperation');
const PersonController = require('../controller/PersonController');


class BasicOperations {
    static main() {
        console.log("Basic Operations! \nAndrea Raura, OOP 14539");
        console.log("--> Unit Testing in action <--");

        let addend1;
        let addend2;
        let sum;

        addend1 = 1.2;
        addend2 = 2.4;
        sum = BasicOperation.addTwoNumbers(addend1, addend2);
        console.log(`The sum of ${addend1} + ${addend2} --> ${sum}`);

        addend1 = 1.3;
        addend2 = 2.5;
        sum = BasicOperation.addTwoNumbers(addend1, addend2);
        console.log(`The sum of ${addend1} + ${addend2} --> ${sum}`);

        addend1 = 1.8;
        addend2 = -2.2;
        sum = BasicOperation.addTwoNumbers(addend1, addend2);
        console.log(`The sum of ${addend1} + ${addend2} --> ${sum}`);

        const personController = new PersonController();
        const bornOnDate = new Date('2004-04-21');
        const days = personController.computeAgeInDays(bornOnDate);
        console.log(`my days are --> ${days}`);

        const numberOfYears = personController.computeAgeInYears(bornOnDate);
        console.log(`my age in years --> ${numberOfYears}`);
    }
}

BasicOperations.main();

File: ws/raura/U2/ws22UnitTesting/BasicOperations/src/main/java/ec/edu/espe/basicoperations/view/BasicOperations.java
package ec.edu.espe.basicoperations.view;

import ec.edu.espe.basicoperations.controller.BasicOperation;
import ec.edu.espe.basicoperations.controller.PersonController;
import java.time.LocalDate;

/**
 *
 * @author Andrea Raura,Paradigm Pioneers Squad, DCCO-ESPE
 */
public class BasicOperations {
    
    public static void main(String[] args){
        System.out.println("Basic Operations! \nAndrea Raura, OOP 14539");
        System.out.println("--> Unit Testing in action <--");
        
        float addend1;
        float addend2;
        float sum;
        
        addend1 = 1.2F;
        addend2 = 2.4F;
        
        sum = BasicOperation.addTwoNumbers(addend1, addend2);
        
        System.out.println("The sum of " + addend1 + " + " + addend2 +" --> " + sum);
        
        addend1 = 1.3F;
        addend2 = 2.5F;
        
        sum = BasicOperation.addTwoNumbers(addend1, addend2);
        
        System.out.println("The sum of " + addend1 + " + " + addend2 +" --> " + sum);
        
        addend1 = 1.8F;
        addend2 = -2.2F;
        
        sum=BasicOperation.addTwoNumbers(addend1, addend2);
        
        System.out.println("The sum of " + addend1 + " + " + addend2 +" --> " + sum);
        
        PersonController personController = new PersonController();
        
        LocalDate bornOnDate;
        bornOnDate = LocalDate.parse("2004-04-21");
        int days;
        
        days = personController.computeAgeInDays(bornOnDate);
        
        System.out.println("my days are --> " + days);
                
        int numberOfYears = personController.computeAgeInYears(bornOnDate);
        
        System.out.println("my age in years --> " + numberOfYears);
        
    
    }

}